### PR TITLE
Confirmation modal

### PR DESCRIPTION
**Pull request by Cafferoq**
### Summary

The Confirmation Modal

### Checklist

- [x] Did you run `npm test`?
- [x] Did you update/add documentation for new methods or changed functionality?

Finally, is it ready to be reviewed and merged: yes :snake: 

### How to Review

1. Pull Consus Client
2. Look at confirm-modal.jsx for possibly poor code
3. On Student.jsx add this code:
```javascript
import ConfirmModal from '../components/confirm-modal.jsx';
```

The following goes in the render function
```javascript
<ConfirmModal
    message="Confirm?"
    active = {true}
    onSelect = {bool => console.log('Conf:', bool)}
/>
```
4. Select student. 
5. Modal will pop up, and if you click a button it will log true or false.
6. ???
7. Profit!

### Links

Put links to Jira tickets, and anything else, here.

- [TFF-113](https://msoese.atlassian.net/browse/TFF-113)

### CODE REVIEW

File: src/views/components/confirm-modal.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Modal from './modal.jsx';
+
+export default class ConfirmModal extends React.Component {
+
+
```
Comment (thofmann) [inline src/views/components/confirm-modal.jsx]: That's a lot of newlines.

File: src/views/components/confirm-modal.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+            <Modal
+                active={this.props.active}
+                buttonText="Yes"
+                onClose={() => this.props.onSelect(true)}>
+                <p>{this.props.message}</p><br/>
+                <br/>
+                <button onClick={() => this.props.onSelect(false)}>No</button>
+            </Modal>
+        )
+    }
```
Comment (thofmann) [inline src/views/components/confirm-modal.jsx]: Any reason for not having a newline here?

Comment (mattmahn) [inline src/views/components/confirm-modal.jsx]: more bytes, probably

File: src/views/components/confirm-modal.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+
+    render() {
+        return (
+            <Modal
+                active={this.props.active}
+                buttonText="Yes"
+                onClose={() => this.props.onSelect(true)}>
+                <p>{this.props.message}</p><br/>
+                <br/>
```
Comment (mattmahn) [inline src/views/components/confirm-modal.jsx]: I think there should only one `<br>` here; looks quite tall with two.

**Pull request merged**